### PR TITLE
feat: print logs of registered providers and consumers

### PR DESCRIPTION
**Pull request by ev1lQuark**
Fix: #2319

### CODE REVIEW

**Review by chickenlj — APPROVED** [review]
LGTM.

Please keep the previous import format unchanged.

**Review by chickenlj — APPROVED** [review]
LGTM.

Please keep the previous import format unchanged.

File: config/consumer_config.go [hunk 1]
```diff
@@ -19,20 +19,15 @@ package config
 
 import (
```
[the remark this answers is not in the document]
Comment (ev1lQuark) [reply config/consumer_config.go:20]: 已恢复

File: config/consumer_config.go [hunk 2]
```diff
@@ -65,12 +66,19 @@ func (cc *ConsumerConfig) Init(rc *RootConfig) error {
 	if cc == nil {
 		return nil
 	}
+
+	consumerNames := make([]string, 0, len(cc.References))
```
[the remark this answers is not in the document]
Comment (ev1lQuark) [reply config/consumer_config.go]: 好的，已经提交了新的 commit，应该能解决这个问题

File: config/provider_config.go [hunk 3]
```diff
@@ -75,14 +76,28 @@ func (c *ProviderConfig) Init(rc *RootConfig) error {
 	if c == nil {
 		return nil
 	}
+	logger.Debug("test")
+	buildDebugMsg := func() string {
```
Comment (bodhili) [inline config/provider_config.go:80]: buildDebugMsg := func() string {
		if len(c.Services) != 0 {
			providerNames := make([]string, 0, len(c.Services))
			for k := range c.Services {
				providerNames = append(providerNames, k)
			}
			return strings.Join(providerNames, ", ")
		} 
		return "empty"
	}

**Review by bodhili — COMMENTED** [review]
remove else

File: config/provider_config.go [hunk 4]
```diff
@@ -75,14 +76,28 @@ func (c *ProviderConfig) Init(rc *RootConfig) error {
 	if c == nil {
 		return nil
 	}
+	logger.Debug("test")
```
Comment (chickenlj) [inline config/provider_config.go:79]: Can this line be removed?

**Review by chickenlj — CHANGES_REQUESTED** [review]

File: config/provider_config.go (hunk 3 above)
Comment (ev1lQuark) [reply config/provider_config.go:79]: OK, I have modified it.

File: config/provider_config.go (hunk 4 above)
Comment (ev1lQuark) [reply config/provider_config.go]: It's my falut, it has been fixed.

**Pull request merged**